### DISCUSSION TITLE
Apache addon usage

### DISCUSSION THREAD

**Discussion by ristoksi** [FaserF/hassio-addons#240] [Q&A] [answered]
Hello! Thanks for the great addon!
What is the preferred way to upload files to the Apache htdocs folder? How to use the mariaDB, should i install phpmyadmin?

**Comment by FaserF** [comment]
Hi,
For MariaDB, please install the official MariaDB HA Addon: 
https://github.com/home-assistant/addons/tree/master/mariadb
My addon integrates with that.

For using files within Apache, your files have to be in the /media or /share folder of HA.
More details can be found in the readme: 
https://github.com/FaserF/hassio-addons/tree/master/apache2

**Reply by ristoksi** [reply]
Thanks. I installed MariaDB and phpmyadmin addons. Also transferred files with fileZilla using SSH.
Do I need to do configurations or should the database work straight? I have set database and user for it and try to select rows from php code, but i get following error:

PHP Fatal error:  Uncaught mysqli_sql_exception: No such file or directory

**Reply by ristoksi** [reply]
Changing servername from localhost to 127.0.0.1 fixes that, but i get new error, connection refused.

**Reply by ristoksi** [reply, marked as answer]
Got it. Servername had to be "core-mariadb".
